### PR TITLE
Eggnog-mapper: make the eggnog data table versionned

### PR DESCRIPTION
**Pull request by abretaud**
Follow-up to #487, trying to fix the versionning of eggnog-mapper.
I feel nervous at touching the data table structure, hope it won't break everything...

### CODE REVIEW

File: tools/eggnog_mapper/eggnog_mapper/eggnog_mapper.xml [hunk 1]
```diff
@@ -42,7 +42,9 @@
     <inputs>
         <param name="input" type="data" format="fasta" label="Fasta sequences to annotate"/>
         <param name="eggnog_data" type="select" label="Version of eggNOG Database">
-            <options from_data_table="eggnog_mapper_db"/>
+            <options from_data_table="eggnog_mapper_db">
```
Comment (bgruening) [inline tools/eggnog_mapper/eggnog_mapper/eggnog_mapper.xml:45]: We probably need to rename the data table to `eggnog_mapper_db_versioned` and the file.

**Review by bgruening — APPROVED** [review]
Great! Thanks @abretaud!

**Comment by abretaud** [comment]
Sorry, I forgot this PR, it should be ready for review/merge now :)

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks @abretaud!

**Pull request merged**